src/components/HeaderWithLanguage/HeaderActions.jsx

```diff
@@ -66,53 +79,58 @@ const HeaderActions = ({
           </HeaderLanguagePill>
         )}
         <StyledSplitter />
```

Comment:
Var litt usikker på hva slags funksjonalitet man trengte å ha i headeren. Hvis den skal være såpass mye enklere enn LanguageHeader som jeg har lagt opp til nå, er det kanskje bedre å lage en ny komponent som SubjectpageHeader?

Reply:
Kan godt tenkes at du bør ha en forenkla komponent for subjectpage i og med at fagsider ikkje har status og heller ikkje oversettelsesfunksjonalitet.